### ISSUE TITLE
Console output not outputting the correct folder paths

### ISSUE THREAD

**Issue by Dan503** [Dan503/gulp-file-loader#7]
This is a note to self issue of a bug I found.

Gulp file portion

```
var sassDest = local_theme_folder+'/sass';

function load_sass(folder) {
	var dest = sassDest+'/'+folder;
	return gulp.src(local_theme_folder+'/sass/'+folder+'/**/*.scss')
		.pipe(fileLoader({ preset: 'scss', dest, fileName: '__includes.scss' }))
		.pipe(gulp.dest(dest))
}

var scssFolders = ['variables', 'mixins', 'components', 'pages'];
var scssLoadTasks = scssFolders.map(folderName => 'load:scss:'+folderName);

scssFolders.forEach((folderName, i) => {
	gulp.task(scssLoadTasks[i], function(){
		return load_sass(folderName)
	})
})

gulp.task('load:scss', scssLoadTasks);
```
console output
```
[10:03:30] Starting 'load:scss:variables'...
[10:03:30] Starting 'load:scss:mixins'...
[10:03:30] Starting 'load:scss:components'...
[10:03:30] Starting 'load:scss:pages'...
gulp-file-loader: Generated ../govcms/sites/all/themes/ipaustralia/sass/pages/__includes.scss
[10:03:30] Finished 'load:scss:variables' after 49 ms
gulp-file-loader: Generated ../govcms/sites/all/themes/ipaustralia/sass/pages/__includes.scss
gulp-file-loader: Generated ../govcms/sites/all/themes/ipaustralia/sass/pages/__includes.scss
[10:03:30] Finished 'load:scss:pages' after 63 ms
[10:03:30] Finished 'load:scss:mixins' after 67 ms
gulp-file-loader: Generated ../govcms/sites/all/themes/ipaustralia/sass/pages/__includes.scss
[10:03:30] Finished 'load:scss:components' after 69 ms
[10:03:30] Starting 'load:scss'...
[10:03:30] Finished 'load:scss' after 33 μs
```

**Comment by Dan503** [comment]
This would be because the dest setting is getting mutated and so the output paths are incorrect as a result of this issue.

**Comment by Dan503** [comment]
Fixed in 2.0.2
